extension/extend-admin-console-spi/README.md

```diff
@@ -0,0 +1,32 @@
+## Extending the admin ui with SPI
+
+This example shows how you can extend the admin ui by implementing the SPI.
+You can either add a tab to an existing page or have a complete section in the menu.
+By implementing either:
+
+  * `org.keycloak.services.ui.extend.UiPageProvider` for a complete section
+  * `org.keycloak.services.ui.extend.UiTabProvider` for a tab
+
+For both options you need to describe the fields you want on the page by using the `org.keycloak.provider.ProviderConfigProperty`
+To specify where the tab will appear in the ui, you can add a path valid paths can be found in the `routes.ts` files.
+
+### Running
+
+Run this example by compiling with maven:
+
+```bash
+$ mvn install
+```
+
+Then copy the generated `extend-admin-ui-1.0-SNAPSHOT.jar` into your <keycloak-server>/provider folder
+### Screenshots
+
+Attribute realm tab:
+![attribute ream tab](images/img.png)
+
+Todo section
+![todo section](images/img_1.png)
+
+Todo detail screen
+![todo detail screen](images/img_2.png)
+
```

Comment:
Please update this text to be more descriptive. It need not be too much, at least add explanations how to get to the admin console from the pictures. Feel free to take [some other quickstart](https://github.com/keycloak/keycloak-quickstarts/blob/6231f33555ff80f4d86372a895efe4497c828242/extension/extend-account-console/README.md?plain=1#L35-L71) as an example.